### PR TITLE
Furkan File Uploaded

### PR DESCRIPTION
**Pull request by durFurkan**
Ready to Review

### CODE REVIEW

File: src/ISIMLER/Furkan.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package ISIMLER;
+
+public class Furkan {
+    public static void main(String[] args) {
+
+        System.out.println("Furkan Local");
```
Comment (zkartal) [inline src/ISIMLER/Furkan.java:6]: abc dogrulamasinida uygula lütfen

Comment (durFurkan) [reply src/ISIMLER/Furkan.java:6]: @zkartal guncellemeler yapildi lutfen inceleyin - https://github.com/Ozleblebici/B11_Git_Practice_Project/pull/6/commits/91a9d9d1089b9e8b3128dd537d6d014f404058ae

Comment (zkartal) [reply src/ISIMLER/Furkan.java:6]: Tesekkürler

**Review by zkartal — APPROVED** [review]
ellerine saglik 10 numara

**Pull request merged**